src/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -145,13 +149,15 @@ export function CurrencyRow({
         onClick={() => (isSelected ? null : onSelect(!!warning))}
         disabled={isSelected}
         selected={otherSelected}
+        dim={dim}
       >
         <Column>
-          <CurrencyLogo currency={currency} size={'36px'} />
+          <CurrencyLogo currency={currency} size={'36px'} style={{ opacity: dim ? '0.6' : '1' }} />
```

Comment:
the opacity level should be a variable since re-used in the component

Reply:
sg done